tests/sentry/utils/test_assets.py

```diff
@@ -28,7 +28,7 @@ def self_hosted(tmp_path: pathlib.Path) -> Generator[None]:
 
 
 @pytest.fixture
-def getsentry_no_configmap(tmp_path: pathlib.Path) -> Generator[None]:
+def frontend_versions_no_configmap(tmp_path: pathlib.Path) -> Generator[None]:
```

Comment:
why do you keep changing things unrelated to the patch

the names here are important -- to represent the `getsentry` case